### ISSUE TITLE
Add "good first issues" to elicit contributions

### ISSUE THREAD

**Issue by activatedgeek** [wilson-labs/cola#13]
One easy way to build a zoo of examples/how-to guides would be to have issues tagged as "Good First Issues", that the authors think would be good additions (and others can create new ones too). I'd be happy to contribute some examples/how-to guides myself once such issues come up.

**Comment by MacOS** [comment]
Same!

**Comment by mfinzi** [comment]
Yeah let's definitely do this now that that the API is a little more stable. 
What might be some good examples, maybe
 - A notebook computing the Estrada index for a large graph using cola.trace(cola.exp(A))
 - An example where a massive MLP is constructed using LinearOperators (say krons, permutations, block_diag of dense randomly initialized matrices) as the learnable weight matrices in an equinox module

And then in terms of functionality
- Hutch++ trace/diagonal estimator
- SVD dispatch rules
- Better numerical methods for SVD base case than Lanczos on A.TA
- Pseudoinverse dispatch rules
- Better numerical methods for Pseudoinverse base case
